assets/src/infrastructure/ui/display/ErrorIndicator.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Alert, AlertIcon, AlertTitle, AlertDescription, IAlert } from '@chakra-ui/core';
+
+interface Props extends IAlert {
+	description?: string;
+	title?: string;
+}
+
+const ErrorIndicator: React.FC<Props> = ({ description, title, ...props }) => {
+	return (
+		<Alert
```

Comment:
plz see my comments about `/infrastructure/ui/display/Alert/Error.tsx`

this is duplicating the same logic as other components in this layer

Reply:
updated `ErrorIndicator`